common/scripted_triggers/hfe_trigger.txt

```diff
@@ -49,6 +51,7 @@ hfe_trigger_is_human_fallen = {
 	or = {
 		has_country_flag = hfe_egypt
 		has_country_flag = hfe_future
+		is_country_type = human_fallen_empire
```

Comment:
redundant

Reply:
No harm, no foul. We simply agree to disagree on redundancy.